src/Settings/SelectInventoryList.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { StyleSheet } from 'react-native';
+import { SafeAreaView } from 'react-navigation';
+import Meteor, { Accounts } from 'react-native-meteor';
+import TableView from 'react-native-tableview'
+const { Section, Item } = TableView;
+
+export default class SelectInventoryList extends Component {
+
+	constructor(props) {
+		super(props);
+
+		const defaultLists = this.props.screenProps.inventoryLists.filter(list => list.isDefault);
+
+		let defaultList;
+		if (defaultLists.length < 1) {
+			defaultList = this.props.screenProps.inventoryLists[0];
+		} else {
+			defaultList = defaultLists[0];
+		}
+
+		this.state = {
+			lists: this.props.screenProps.inventoryLists,
+			defaultList
+		};
+
+	}
+
+	static navigationOptions({ navigation }) {
+		return {
+			headerTitle: 'Default Inventory List'
+		};
+	}
+
+	setDefaultList(item) {
+		const inventoryList = this.state.lists[item.selectedIndex];
+
+		// remove the old default list then set a new one
+		Meteor.call('inventorylists.setDefault', this.state.defaultList._id, false);
```

Comment:
Ah, instead of making multiple calls, should we just make it that when you call `inventorylists.setDefault`, it automatically checks that there are no other lists already marked as the default, and if so, it will make that other list not the default anymore?

That, or, we just add additional checks to make sure that you can't have two lists marked as the default

Reply:
Yeah good point. I'll try to add that now. I know I thought of something similar around list creation (https://trello.com/c/siBp3XEt) but it should also be done here.